docs/theory/force_calibration/force_calibration.rst

```diff
@@ -0,0 +1,357 @@
+However, there are some practical aspects to consider.
+So far, we have only considered the expectation value of the power spectrum.
+In reality, power spectral values follow a distribution.
+
+The real and imaginary part of the frequency spectrum are normally distributed.
+As a consequence, the squared magnitude of the power spectrum is exponentially distributed.
+This has two consequences:
+
+- Fitting the power spectral values directly using a simple least squares fitting routine, we would get very biased estimates.
+- The signal to noise ratio is poor (equal to one :cite:`norrelykke2010power`).
+
+A commonly used method for dealing with this involves data averaging, which trades resolution for an improved signal to noise ratio.
+In addition, by virtue of the central limit theorem, data averaging leads to a more symmetric data distribution (more amenable to standard least-squares fitting procedures).
+
+There are two ways to perform such averaging:
+
+- The first is to split the time series into windows of equal length, compute the power spectrum for each chunk of data and averaging these. This procedure is referred to as windowing.
+- The second is to calculate the spectrum for the full dataset and downsampling the obtained spectrum in the spectral domain by averaging adjacent bins according to :cite:`berg2004power`. This procedure is referred to as blocking.
+
+We use the blocking method for spectral averaging.
+This allows us to reject noise peaks at high resolution prior to blocking.
+Note however, that the error incurred by this blocking procedure depends on :math:`n_b`, the number of points per block, :math:`\Delta f`, the spectral resolution and inversely on the corner frequency :cite:`berg2004power`.
+
+Setting the number of points per block too low would result in a bias from insufficient averaging :cite:`berg2004power`.
+In practice, one should use a high number of points per block (:math:`n_b >> 100`), unless a very low corner frequency precludes this.
+In such cases, it is preferable to increase the measurement time.
+
+Passive calibration
+-------------------
+
+Passive calibration is also often referred to as thermal calibration and involves calibration without moving the trap or stage.
+In passive calibration, the Brownian motion of the bead in the trap is analyzed in order to find calibration factors for both the positional detection as well as the force.
+
+To fit passive calibration data, we will use a model based on a number of publications by the Flyvbjerg group :cite:`berg2004power,tolic2004matlab,hansen2006tweezercalib,berg2006power`.
+This model can be found in :class:`~.PassiveCalibrationModel`.
+The most basic form of passive calibration starts by fitting the following equation to the power spectrum:
+
+.. math::
+
+    P(f) = \frac{D_\mathrm{measured}}{\pi ^ 2 \left(f^2 + f_c ^ 2\right)} g(f, f_\mathrm{diode}, \alpha) \quad \mathrm{[V^2/Hz]}
```

Comment:
Hmm. Personally, I quite like having them there, since it makes it more explicit what is in what units (which is important since we're talking about calibration factors here). I wonder if we can improve it with some formatting changes. Though I wouldn't know what.